### ISSUE TITLE
Minor runtime issues

### ISSUE THREAD

**Issue by jmansour** [barbagroup/geoclaw-landspill-cases#20]
Hi guys

Just a few minor suggestions as a follow up on the JOSS review process. 

1. On my system (Ubuntu 20.04 via Windows WSL), the runtime thread count defaults to the _logical_ processors count, instead of the physical core count. That's effectively oversubscribing the system, as hyper-threading is generally not useful for these problem classes.  If I instead explicitly set `OMP_NUM_THREADS=4` (the number of physical cores), the job runs around 25% faster when compared to the same job without setting `OMP_NUM_THREADS` (which defaults to 8 threads).  I actually prefer as a default that a single thread be used, with the user then required to explicitly configure their environment to use more threads. But otherwise using the physical core count is also a reasonable default.
2. Is there a runtime parameter that adjusts the standard outputting verbosity level? The default the outputting seems somewhat excessive.

**Comment by piyueh** [comment]
Hi @jmansour, thanks for the review! It's great to hear this project also runs under WSL! We don't have much experience with WSL, so it's great to know this.

1. Yes, from our experience, most of the time it's better to use only physical cores. I think we can set the default OpenMP thread number to the number of physical cores in the Python launcher `geoclaw-landspill`. And users can control the number of threads through a CMD parameter if they want. I'll leave this issue open until we add this feature.

2. Currently, the verbosity of log output is set in the `setrun.py` script in each simulation case. The corresponding parameter is `rundata.clawdata.verbosity`. By default it's 5, meaning all information of AMR mesh of level 5 and lower will be printed out. Changing it to 0 disables the logging, and changing it to 1 only prints the info of the coarsest mesh. I think it's nice to have a CMD parameter in `geoclaw-landspill` to control the verbosity as well. Then there's no need to edit `setrun.py` just to change the log verbosity. Will leave this issue open until done.

**Comment by piyueh** [comment]
(Just to to add a link to the reviewer's comment at https://github.com/openjournals/joss-reviews/issues/3114#issuecomment-827332168)

**Comment by piyueh** [comment]
Hi @jmansour I have added the features to the code. They are now available in the current master branch with tag [v1.0.dev4](https://github.com/barbagroup/geoclaw-landspill/tree/v1.0.dev4).

1. The default number of OpenMP threads is now the number of physical cores. Added in the commit [784c7d70](https://github.com/barbagroup/geoclaw-landspill/commit/784c7d7026dd278da7479a8f07c60630214ca0a6).
2. The logging verbosity can now be changed with CMD flag `--log-level <level>`. Added in the commit [989559e](https://github.com/barbagroup/geoclaw-landspill/commit/989559e5d8159c87ab8f8de3121a2f23bb905e52). However, it only turns off the log messages of iterations and regridings. It does not disable the warnings, for example, CFL warnings. Those warnings can not be disabled without modifying the Fortran code. So for simulations with rapidly changing flow conditions, the standard output may still be overwhelmed by those warning messages.
